opentracing-api/src/main/java/io/opentracing/ActiveSpanSource.java

```diff
@@ -39,12 +38,18 @@
     ActiveSpan activeSpan();
 
     /**
+     * @see #makeActive(Span, ActiveSpan.Observer)
+     */
+    ActiveSpan makeActive(Span span);
+    /**
      * Wrap and "make active" a {@link Span} by encapsulating it – and any active state (e.g., MDC state) in the
      * current thread – in a new {@link ActiveSpan}.
      *
      * @param span the Span to wrap in an {@link ActiveSpan}
+     * @param observer the observer that learns of activations, deactivations, and captures related to the returned
+     *                 {@link ActiveSpan}. May be null (which implies "no observer")
      * @return an {@link ActiveSpan} that encapsulates the given {@link Span} and any other
      *     {@link ActiveSpanSource}-specific context (e.g., the MDC context map)
      */
-    ActiveSpan makeActive(Span span);
+    ActiveSpan makeActive(Span span, ActiveSpan.Observer observer);
```

Comment:
One idea (not saying it's a good one... or that I understand all the implications) for making this be more backwards compatible is to make the `Observer` argument be a var arg.  This would have the additional side effect of potentially allowing multiple `Observer`s.